### PR TITLE
Note on String.p.includes case sensitivity

### PR DESCRIPTION
**Pull request by manngo**
Added a note about emulating case sensitivity by changing the case of the original string.

#### Summary

Added using `toLowerCase()` to emulate case sensitivity.
#### Motivation

To answer the begging question “so how do I …?”
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error
- [x] Adds a minor point.

### CODE REVIEW

**Comment by manngo** [comment]
Is it too late to change

	However, you emulate …

to

	However, you can emulate …

?

**Comment by teoli2003** [comment]
You can, this has not yet been reviewed.

File: files/en-us/web/javascript/reference/global_objects/string/includes/index.md [hunk 1]
```diff
@@ -51,6 +51,12 @@ expression returns `false`:
 'Blue Whale'.includes('blue')  // returns false
 ```
 
+However, you emulate a case insensitive match by changing the case of the original string:
```
Comment (teoli2003) [inline files/en-us/web/javascript/reference/global_objects/string/includes/index.md:54]: What about:
```suggestion
You can work around this constraint by changing the case of the string:
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
We are close to be able to merge this (sorry for the delay, I missed the notification)

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**